README.md

```diff
@@ -0,0 +1,49 @@
+The [@devcontainers-community]'s mission is to be a **centralized source** for
+**well-maintained** features that are of **high-quality** similar to those of
+the official [@devcontainers] organization, but for more **popular tools and
+languages**.
+
+We want to avoid atrophy by investing into the _community-based_ approach to
+maintenance that has been successful in other communtiy orgainzations like
+[@asdf-community]. This means making it as easy as possible to contribute. 🤝
+
+This is, however, a _fallback option_ for languages and tools that do not have
+official awesome-tool/devcontainer-features or related repositories to
+self-maintain within that community. If you are an individual who is part of a
+popular tool, consider [authoring your own feature repository]! ❤️
+
+If you're interested in joining this organization, that's awesome! 🤩 Open a
+Discussion and let's chat! If you just want to add a new feature, check out our
+[feature contribution guide]! 🌟
+
+## Scope
+
+Here's some specific examples of things that are in scope for this organization:
+
+- Maintaining popular language features, images, and templates that
+  [@devcontainers] does not cover
```

Comment:
```suggestion
- Maintaining popular language Dev Container Features, and Dev Container Templates that
  [@devcontainers] does not cover
```

Since maintaining images is very effort intensive, I would recommend starting with only Features and Templates.
If we distribute Docker images, we will need to start to create the build system (like I did in https://github.com/rocker-org/devcontainer-images)

Reply:
Good idea! I like it.